pkg/utils/utils.go

```diff
@@ -77,7 +84,7 @@ func escapeToJSONstringFormat(line string) (string, error) {
 
 // ExecuteCommand uses the generic command handler to execute an arbitrary interactive command, returning
 // its output wihout any other check.
-func ExecuteCommand(command string, timeout time.Duration, context *interactive.Context, failureCallbackFun func()) string {
+func ExecuteCommand(command string, timeout time.Duration, context *interactive.Context, failureCallbackFun func(), runopt string) string {
```

Comment:
I don't think this function should have extra parameters, but if it's necessary. It's better to create an Enum and pass that Enum. Right now, you're passing a string, and not using it, you only care if it's empty or not.